src/binder/bind/bind_export_database.cpp

```diff
@@ -57,25 +59,29 @@ static void bindExportNodeTableDataQuery(const TableCatalogEntry& entry, std::st
 }
 
 static void bindExportRelTableDataQuery(const TableCatalogEntry& entry, std::string& exportQuery,
-    const catalog::Catalog& catalog, transaction::Transaction* tx) {
+    const Catalog& catalog, const Transaction* transaction) {
     auto relTableEntry = entry.constPtrCast<RelTableCatalogEntry>();
-    auto srcPrimaryKeyName = getPrimaryKeyName(relTableEntry->getSrcTableID(), catalog, tx);
-    auto dstPrimaryKeyName = getPrimaryKeyName(relTableEntry->getDstTableID(), catalog, tx);
-    auto srcName = catalog.getTableCatalogEntry(tx, relTableEntry->getSrcTableID())->getName();
-    auto dstName = catalog.getTableCatalogEntry(tx, relTableEntry->getDstTableID())->getName();
+    auto srcPrimaryKeyName =
```

Comment:
I think you should first get src entry here and then get primary key name and table name from the entry.

Reply:
Make sense.